### PR TITLE
(Feat/VVS-CPC)695_insert_photo

### PR DESCRIPTION
**Pull request by Brandon84-alt**
JIRA: link to jira ticket
https://champlainsaintlambert.atlassian.net/browse/CPC-695
What is the ticket about and why are we doing this change.
-Enables Vet to Uplaod Photo on Click/Hover on Photo Icon.
-Should Upload a photo
-Default Photo should appear on hover in Vets Form
 
What are the various changes and what other modules do those changes effect.
This can be bullet point or sentence format.

Before and After UI (Required for UI-impacting PRs)
-When you over on Vet Form you can see vet Speciatlty and Default photo (null for the moment)

Dev notes (Optional)
Photo Fix to assign to VetId is coming in branch 694_Fix_Photo_for_Vet

### CODE REVIEW

File: api-gateway/src/main/resources/static/scripts/vet-details/vet-details.controller.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 angular.module('vetDetails')
     .controller('VetDetailsController', ['$http', '$stateParams', function ($http, $stateParams) {
         var self = this;
+        //var vetId = $stateParams.vetId || 0;
```
Comment (Kam1st) [inline api-gateway/src/main/resources/static/scripts/vet-details/vet-details.controller.js:6]: Let's not forget to take out the commented code we have, just to make it clean.

File: vet-service/src/main/java/com/petclinic/vet/dataaccesslayer/Photo.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+//package com.petclinic.vet.dataaccesslayer;
```
Comment (Kam1st) [inline vet-service/src/main/java/com/petclinic/vet/dataaccesslayer/Photo.java:1]: Is this class necessary?

**Review by Kam1st — APPROVED** [review]
Going to have to check out the commented code to see what there is to keep/get rid of.

File: vet-service/src/main/java/com/petclinic/vet/dataaccesslayer/Photo.java (hunk 2 above)
Comment (Brandon84-alt) [reply vet-service/src/main/java/com/petclinic/vet/dataaccesslayer/Photo.java:1]: For my second story (694) which is connected to this user storie will need this package for photos implementation in Backend.

File: api-gateway/src/main/resources/static/scripts/vet-details/vet-details.controller.js (hunk 1 above)
Comment (Brandon84-alt) [reply api-gateway/src/main/resources/static/scripts/vet-details/vet-details.controller.js:6]: Will also be needed to have a vetId variable declare for user storie (694) on Joira

**Review by cubajosh — APPROVED** [review]

**Pull request merged**